specification/communication/data-plane/Rooms/preview/2023-03-31-preview/communicationservicesrooms.json

```
@@ -72,6 +72,42 @@
             "$ref": "./examples/Rooms_CreateRoom.json"
           }
         }
+      },
+      "get": {
+        "tags": [
+          "Rooms"
+        ],
+        "summary": "Retrieves all created rooms.",
+        "operationId": "Rooms_GetRooms",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "Request successful. The action returns a collection of rooms.",
+            "schema": {
+              "$ref": "#/definitions/RoomsCollection"
+            }
+          },
+          "default": {
+            "description": "Error response",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "x-ms-client-name": "ErrorCode",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get meeting room.": {
+            "$ref": "./examples/Rooms_GetRooms.json"
```

Let's also rename the example file.

```suggestion
            "$ref": "./examples/Rooms_ListRooms.json"
```